Code/Framework/AzToolsFramework/AzToolsFramework/Editor/ActionManagerIdentifiers/EditorActionUpdaterIdentifiers.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/string/string_view.h>
+
+namespace EditorActionUpdater
+{
+    static constexpr AZStd::string_view AngleSnappingStateChangedUpdaterIdentifier = "o3de.updater.onAngleSnappingStateChanged";
```

Comment:
update: I believe you can omit `static` here (and `inline` should be preferred if anything but I believe `constexpr` already implies this

Reply:
Good point, will do.

Reply:
Looks like `constexpr` does not imply `inline` for non-static variables. Would it make sense to write `inline constexpr`?

Reply:
(Watched some videos and read some docs, it makes sense to use `inline constexpr` for variables in header space. Doing that!)